sdk/storage/azure-storage-blob/azure/storage/blob/_lease.py

```diff
@@ -42,7 +41,6 @@ class BlobLeaseClient(): # pylint: disable=client-accepts-api-version-keyword
     last_modified: Optional["datetime"]
     """The last modified timestamp of the lease currently being maintained.
     This will be `None` if no lease has yet been acquired or modified."""
-    _client: Union["BlobOperations", "ContainerOperations"]
```

Comment:
I figured it out. This is the only client where we annotated the internal `_client`, and thus MyPy was too smart for it's own good and hurt itself in confusion. By omitting this, this makes the generated code call undiscoverable, thus not enforcing type checking on the return 😄